packages/gatsby-theme-carbon/gatsby-node.mjs

```diff
@@ -0,0 +1,120 @@
+import path from 'path';
```

Comment:
I am not sure why git renamed gatsby-config.js to gatsby-config.mjs and is adding this as a new file after removing the older one.

Reply:
You probably need to set `"type:" "module"` in the **package.json** so that the entire project defaults to modules instead of CommonJS. Give that a try and see if you can rename the files back to just .js

Reply:
To be honest I was just trying to do that only. But I am getting this - 
`
 ERROR #10123  API.CONFIG.LOADING

We encountered an error while trying to load your site's gatsby-config. Please fix the error and try again.



  ReferenceError: require is not defined in ES module scope, you can use import instead
  
  - gatsby-config.mjs:1 
    file:///Users/mjuneja/Work/mine/OS/gatsby-theme-carbon/packages/gatsby-theme-carbon/gatsby-config.mjs:1:14
  
  - module_job:194 ModuleJob.run
    node:internal/modules/esm/module_job:194:25
  
`